src/Servers/Kestrel/Transport.Sockets/src/SocketTransportOptions.cs

```diff
@@ -65,6 +67,16 @@ public class SocketTransportOptions
         /// </remarks>
         public bool UnsafePreferInlineScheduling { get; set; }
 
+        /// <summary>
+        /// An action used to configure the listening socket before it is bound.
+        /// </summary>
+        public Action<EndPoint, Socket>? ConfigureListenSocket { get; set; }
+
+        /// <summary>
+        /// An action used to configure an accept socket before it's passed to the underlying connection.
+        /// </summary>
+        public Action<EndPoint, Socket>? ConfigureAcceptSocket { get; set; }
```

Comment:
Would `ConfigureAcceptedSocket` be clearer?

Reply:
@Tratcher no objections here, will update it tomorrow morning 👍